front/app/components/IdeaCards/IdeasWithFiltersSidebar/index.tsx

```diff
@@ -230,6 +231,14 @@ const IdeasWithFiltersSidebar = ({
               ),
             })}
           </Title>
+        ) : (
+          <Box ml="auto">
+            <Text m="0px">
+              {formatMessage(messages.numberResults, {
+                postCount: list ? list.length : 0,
```

Comment:
I want to deal with the `list` check in this component as well when cleaning up. You shouldn't have to do that here. But ok to merge for now, since it'll cause conflicts with other work I'm doing.